### ISSUE TITLE
EXC_BAD_ACCESS (SIGBUS) in ru.yandex.mobile.YandexMobileMetrica.YMMReporter.ivarQueue thread

### ISSUE THREAD

**Issue by wacumov** [yandexmobile/metrica-sdk-ios#38]
Hello. I've received a crash report via AppMetrica. I found that YandexMobileMetrica thread was caused the crash. Please, look at the attached file.

Exception Type:  EXC_BAD_ACCESS (SIGBUS)
Exception Codes: 0x00000000 at 0x0000000000000050
Crashed Thread:  7

Thread 7 name:  Dispatch queue: ru.yandex.mobile.YandexMobileMetrica.YMMReporter.ivarQueue
Thread 7 Crashed:
0   libdispatch.dylib               0x000000018146e668 0x18146c000 + 9832 (dispatch_sync + 0)
1   AppSupport                      0x00000001832205a0 0x1831f4000 + 181664 (CPLoggingAddCustomLogConfiguration + 1740)
2   ManagedConfiguration            0x000000018477898c 0x184770000 + 35212 (<redacted> + 220)
3   libdispatch.dylib               0x000000018146d47c 0x18146c000 + 5244 (<redacted> + 16)
4   libdispatch.dylib               0x000000018146e2c0 0x18146c000 + 8896 (dispatch_once_f + 80)
5   ManagedConfiguration            0x000000018477a9d8 0x184770000 + 43480 (<redacted> + 40)
6   ManagedConfiguration            0x000000018477c2dc 0x184770000 + 49884 (<redacted> + 92)
7   AdSupport                       0x000000018c72cd64 0x18c72c000 + 3428 (<redacted> + 32)
8   filki                           0x00000001000d95cc 0x100004000 + 873932 (_mh_execute_header + 873932)
9   filki                           0x00000001000ff250 0x100004000 + 1028688 (_mh_execute_header + 1028688)
10  filki                           0x00000001000fed88 0x100004000 + 1027464 (_mh_execute_header + 1027464)
11  filki                           0x00000001000f6bcc 0x100004000 + 994252 (_mh_execute_header + 994252)
12  filki                           0x00000001000f6a3c 0x100004000 + 993852 (_mh_execute_header + 993852)
13  filki                           0x00000001000f6114 0x100004000 + 991508 (_mh_execute_header + 991508)
14  libdispatch.dylib               0x000000018146d4bc 0x18146c000 + 5308 (<redacted> + 24)
15  libdispatch.dylib               0x000000018146d47c 0x18146c000 + 5244 (<redacted> + 16)
16  libdispatch.dylib               0x00000001814794c0 0x18146c000 + 54464 (<redacted> + 864)
17  libdispatch.dylib               0x0000000181470f80 0x18146c000 + 20352 (<redacted> + 464)
18  libdispatch.dylib               0x000000018146d47c 0x18146c000 + 5244 (<redacted> + 16)
19  libdispatch.dylib               0x000000018147b914 0x18146c000 + 63764 (<redacted> + 2140)
20  libdispatch.dylib               0x000000018147b0b0 0x18146c000 + 61616 (<redacted> + 112)
21  libsystem_pthread.dylib         0x0000000181685470 0x181684000 + 5232 (_pthread_wqthread + 1092)

[AppMetrica.txt](https://github.com/yandexmobile/metrica-sdk-ios/files/267726/AppMetrica.txt)

**Comment by yvasileuski** [comment]
Is it possible to provide symbolicated crash?

**Comment by Ewg777** [comment]
The same here
```Crashed: ru.yandex.mobile.YandexMobileMetrica.YMMReporter.ivarQueue
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000020

 Raw Text
0	CoreFoundation	CFBundleGetFunctionPointerForName + 29
1	MYAPP	-[YMMFramework functionFromString:] + 171074
2	MYAPP	-[YMMFramework functionFromString:] + 171074
3	MYAPP	+[YMMWiFiNetworkProvider loadFrameworkIfNeeded] + 400190
4	MYAPP	+[YMMWiFiNetworkProvider currentNetworks] + 400422
5	MYAPP	+[YMMNetworkInfo networkInfoWithReachabilityStatus:] + 252570
6	MYAPP	+[YMMNetworkInfo currentNetworkInfo] + 252438
7	MYAPP	-[YMMEventBuilder eventWithClass:ofType:appEnvironment:] + 158344
8	MYAPP	-[YMMEventBuilder eventAliveWithAppEnvironment:] + 158074
9	MYAPP	-[YMMReporter endSession:updatedAt:] + 274734
10	MYAPP	-[YMMReporter ensureSessionEnded:] + 274358
11	MYAPP	 __20-[YMMReporter setup]_block_invoke + 272192
12 libdispatch.dylib	_dispatch_call_block_and_release + 10
13	libdispatch.dylib	_dispatch_queue_drain$VARIANT$mp + 1758
14	libdispatch.dylib	_dispatch_queue_invoke$VARIANT$mp + 284
15	libdispatch.dylib	_dispatch_root_queue_drain + 1560
16	libdispatch.dylib	_dispatch_worker_thread3 + 96
17	libsystem_pthread.dylib	_pthread_wqthread + 1024
18 libsystem_pthread.dylib	start_wqthread + 8
```

**Comment by yvasileuski** [comment]
Fixed in Version 2.9.0
